README.md

```diff
@@ -95,10 +97,51 @@ bothers our programmer brains about this. It's seeing _so much repetition_.
 
 Could we reduce the repetition somehow?
 
-If we created a method to contain this "saying `message` five times action" we
-could get rid of some of that repetition.
+If we created a _method_ to contain this "saying `message` five times action" we
+could get rid of some of that repetition. Let's get to it! First, let's look at
+the "template" of a method:
+
+## Produce the Shell of a Method Definition
+
+```ruby
+def name_of_method(parameter1, parameter2, ... parameter_n)
+# Code goes here
+end
+```
+
+A method is defined by using the Ruby language word (or "keyword") `def`
+followed by a name of our choosing.  The first line of `def name_of_method` is
+called the method _signature_, it defines the basic properties of the method
+including the name of the method, `name_of_method`. A method's name should
+begin with a lowercase letter.  If you need multiple words, Rubyists use a `_`
+to separate them. Separating words by underscore (`_`) is called _snake-case_
+(because the shape looks like the words were `swallowed_up_by_a_snake`).
+
+We'll ignore the parameters for the time being (but they'll come momentarily!).
+Lastly we have to provide the keyword `end`.
+
+Between the `def` and the `end` we write our _implementation_ or _method body_.
+
+## Define "Implementation"
+
+The implementation is the "more atomic" evaluations and statements that do the
+work implied by the method's name. Just like `make_dinner` includes all sorts
```

Comment:
sounds like referring to a previous example. Maybe switch to "making dinner" to make it clearer that we're talking about the actual process we call "making dinner"